docker-app/qfieldcloud/core/serializers.py

```diff
@@ -535,3 +546,44 @@ class LatestPackageSerializer(serializers.Serializer):
     package_id = serializers.UUIDField()
     packaged_at = serializers.DateTimeField()
     data_last_updated_at = serializers.DateTimeField()
+
+
+class TeamSerializer(serializers.ModelSerializer):
+    organization = serializers.StringRelatedField(source="team_organization")
+
+    class Meta:
+        model = Team
+        fields = ("username", "organization")
+
+
+class TeamMemberSerializer(serializers.ModelSerializer):
+    member = serializers.CharField()
+
+    def to_internal_value(self, data: Dict[str, Any]) -> Dict[str, Any]:
+        validated_data = super().to_internal_value(data)
+        email_or_username = data.get("member")
+
+        try:
+            existing_user = User.objects.fast_search(email_or_username)
+
+            if TeamMember.objects.filter(member=existing_user):
+                raise serializers.ValidationError(
+                    {
+                        "member": f"A member with username '{email_or_username}' already exists."
```

Comment:
```suggestion
                        "member": _('Team member "{}" already exists.').format(email_or_username)
```

IMO the messages from the `ValidationError` should be translated. It might not be the case already in most of the code, but this is the direction.

Reply:
So, The _() function marks strings for translation, correct? Thus, it makes the string translatable to other languages.. Fair enough!